lib/USAGE.md

```diff
@@ -64,20 +86,24 @@ jws.public_key = public_key
 
 # JWS will verify using the included public_key
 jws.verify
+
+health_card = HealthCards::HealthCard.new(jws)
+health_card.verify
```

Comment:
Does this method actually work? Verifying requires a public key which `Verifier`'s have, but HealthCards themselves don't.

We'd at least need to have `verify` accept a key or keyset if we wanted to keep this functionality.

Reply:
We discussed this offline, but basically it delegates to the internal JWS. If the the JWS is from an external source (and we don't have the source public key), it'll use remote key resolution. Otherwise, it'll use the key contained in the JWS object.

I'll update `Verifier` so that it can accept a `HealthCard`, in addition to a `JWS`